compiler/sem/semexprs.nim

```diff
@@ -3276,4 +3276,10 @@ proc semExpr(c: PContext, n: PNode, flags: TExprFlags = {}): PNode =
   of nkError: discard "ignore errors for now"
   else:
     localReport(c.config, n, reportSem rsemInvalidExpression)
-  if result != nil: incl(result.flags, nfSem)
+
+  if result != nil:
+    if c.config.isCompilerDebug():
```

Comment:
Personally not a fan of this over simply using and

Reply:
This is a random hack for debugging I added to get some real AST to print